### PR TITLE
Add build-installers step to CI

### PR DESCRIPTION
**Pull request by igor-hnizdo**
Note that this takes about 25 minutes to build, every time, because docker on github does not use any caching; and some things are really slow (downloading all the musl libs, compiling Rust code, all the dnf updates).

We can use this thing below to improve the times, but I wonder if it's worth it adding some random action that overrides docker behavior. Probably not.

https://github.com/marketplace/actions/docker-layer-caching

### CODE REVIEW

**Comment by vdovhanych** [comment]
This builds packages without signing them, right? Let's maybe add the build output as artifacts. Would that be useful for testing purposes? @tsusanka

**Comment by tsusanka** [comment]
> Let's maybe add the build output as artifacts. Would that be useful for testing purposes? @tsusanka

Yes, definitely could you add it @igor-hnizdo?

> We can use this thing below to improve the times, but I wonder if it's worth it adding some random action that overrides docker behavior. Probably not.

Yeah, let's not worry about that, the development here is very rare, so it is enough.

**Comment by igor-hnizdo** [comment]
OK. We can do that definitely.

Yes, this builds all the binaries and all the packages. They will show warnings on macOS and Windows on install, but will work in general.

**Comment by vdovhanych** [comment]
Sure, id add it as artifacts fe.
```yml
      - name: Upload build results
        uses: actions/upload-artifact@v3
        with:
          name: trezord-go_bin
          path: |
          	- path to file
          	- path to file
```

**Comment by igor-hnizdo** [comment]
I just did that, a bit differently from you (I made 2 steps for the 2 folders)

**Comment by vdovhanych** [comment]
cc @tsusanka Do we need to have binaries? Wouldn't the installer be enough for testing?

**Comment by igor-hnizdo** [comment]
Well the installer is enough, but ... I guess sometimes you want to test just the binary? So you don't need to go through uninstall-reinstall cycle

File: .github/workflows/check-build-installers.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: "[Check] Build installers without certs"
```
Comment (vdovhanych) [inline .github/workflows/check-build-installers.yml:1]: Also I would name this a bit differently
```suggestion
name: "[Build] unsigned installers and binaries"
```

Comment (igor-hnizdo) [reply .github/workflows/check-build-installers.yml]: ok

**Review by tsusanka — APPROVED** [review]
Awesome.

**Comment by tsusanka** [comment]
I have fixed some typos in https://github.com/trezor/trezord-go/commit/afce7a3ecb556e0bf00f0648933d12ea3de35414. With an incorrect commit message 🙄.

**Pull request merged**